custom_components/daikin_onecta/update.py

```
@@ -0,0 +1,159 @@
+# point.  The relevant characteristics are:
+#   firmwareVersion       – currently installed version string
+#   isFirmwareUpdateSupported – whether the gateway hardware supports OTA updates
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Daikin update entities from a config entry."""
+    onecta_data: OnectaRuntimeData = config_entry.runtime_data
```

Ruff (0.15.12)

[error] Undefined name `OnectaRuntimeData`

(F821)